config/retry/backoff.go

```diff
@@ -218,7 +218,12 @@ func (b *Backoffer) BackoffWithCfgAndMaxSleep(cfg *Config, maxSleepMs int, err e
 	}
 
 	if b.vars != nil && b.vars.Killed != nil {
-		if atomic.LoadUint32(b.vars.Killed) == 1 {
+		killed := atomic.LoadUint32(b.vars.Killed)
+		if killed != 0 {
+			logutil.BgLogger().Info(
+				"backoff stops because a killed signal is received",
+				zap.Uint32("signal", killed),
+			)
```

Comment:
How about wrapping the kill signal check into a function inside the `Backoffer` type.
Besides, we need to make sure this method is used by all backoff types.

Reply:
All backoff methods go into `BackoffWithCfgAndMaxSleep` which checks the kill signal. Is this what you meant?

Reply:
Yes